stdlib/arg.ml

```diff
@@ -117,9 +117,7 @@ let usage speclist errmsg =
 
 let current = ref 0
 
-let bool_of_string_opt x =
-  try Some (bool_of_string x)
-  with Invalid_argument _ -> None
+let bool_of_string_opt = Bool.of_string
```

Comment:
Let's drop this definition and inline it on its single call site (and yes, the same should be done for the two later definitions, but it's unrelated to this PR).